README.md

```diff
@@ -16,3 +16,60 @@ $> cp sample.env .env
 ```
 $> npm start
 ```
+
+## Localization
+
+In this project we're using [React-Intl](https://github.com/yahoo/react-intl) to localize our application and YAML for translation.
+
+#### Localize a component or page
+
+To localize a component or page you have to always include IntlMixin in your class mixins for example:
+
+``` typescript
+var React = require('react');
+var IntlMixin = require('react-intl').IntlMixin;
+
+var Example = React.createClass({
+	mixins: [IntlMixin]
+  render: function() {
+    return (
+      <div>
+      	<h1>{this.getIntlMessage('key_name_here')}
+      </div>
+    );
+  }
+
+});
+```
+
+Once you added the mixin it will expose `getIntlMessage` method for your component to get the localized message for the given key name.
```

Comment:
Reading this from the top down, I'm now mostly OK with this bit, I just have tiny changes now.

"Once you add the mixin it will expose `getIntlMessage` method to your component to get the localized message for the given key"

I changed added/add for/to and removed name